lib/resque/worker.rb

```diff
@@ -359,7 +408,7 @@ def startup
       register_signal_handlers
       start_heartbeat
       prune_dead_workers
-      run_hook :before_first_fork
+      run_hook :before_first_fork, self
```

Comment:
FEATURE: pass `self` to `before_first_fork` callbacks.

This is used by the monolith to pause processing if an FS worker starts up in the wrong place.

This could be PRed to master, existing users won't be impacted by an unexpected arg.